packages/aila/src/core/document/AilaDocument.test.ts

```diff
@@ -0,0 +1,274 @@
+import { aiLogger } from "@oakai/logger";
+import type { z } from "zod";
+
+import type { Message } from "../chat";
+import { LessonPlanSchema } from "./schemas/lessonPlan";
+import type { AilaDocumentContent, CategorisationPlugin } from "./types";
+
+const log = aiLogger("aila");
+
+class TestDocument {
```

Comment:
This file could do with a comment explaining what it does, as as far as I can tell it's defining a local class which isn't used in the codebase (yet?) and then testing it